packages/clerk-js/src/utils/authPropHelpers.ts

```diff
@@ -112,3 +112,17 @@ export const buildAuthQueryString = (data: BuildAuthQueryStringArgs): string | n
   }
   return Object.keys(query).length === 0 ? null : qs.stringify(query);
 };
+
+export const normalizeRoutingOptions = ({
```

Comment:
🙃 This file is not just about `authPropHelpers` anymore. It is a collection of prop/options parsers and normalizers. So, at some point, we need to rename it.

Reply:
Noted!